samples build fail due to checkstyle violations

#### Steps to reproduce

  mvn -P lint --quiet --batch-mode checkstyle:check

```
[ERROR] src\main\java\com\example\firestore\Quickstart.java:[132,8] (naming) AbbreviationAsWordInName: Abbreviation in name 'runAQuery' must co
ntain no more than '1' consecutive capital letters.
[ERROR] src\main\java\com\example\firestore\snippets\ManageDataSnippets.java:[448,29] (whitespace) WhitespaceAround: WhitespaceAround: ':' is n
ot followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)
[ERROR] src\main\java\com\example\firestore\snippets\QueryDataSnippets.java:[114,9] (naming) AbbreviationAsWordInName: Abbreviation in name 'cr
eateAQuery' must contain no more than '1' consecutive capital letters.
[ERROR] src\main\java\com\example\firestore\snippets\QueryDataSnippets.java:[137,9] (naming) AbbreviationAsWordInName: Abbreviation in name 'cr
eateAQueryAlternate' must contain no more than '1' consecutive capital letters.
[ERROR] src\main\java\com\example\firestore\snippets\References.java:[37,30] (naming) AbbreviationAsWordInName: Abbreviation in name 'getAColle
ctionRef' must contain no more than '1' consecutive capital letters.
[ERROR] src\main\java\com\example\firestore\snippets\References.java:[52,28] (naming) AbbreviationAsWordInName: Abbreviation in name 'getADocum
entRef' must contain no more than '1' consecutive capital letters.
[ERROR] src\main\java\com\example\firestore\snippets\References.java:[67,28] (naming) AbbreviationAsWordInName: Abbreviation in name 'getADocum
entRefUsingPath' must contain no more than '1' consecutive capital letters.
[ERROR] src\main\java\com\example\firestore\snippets\References.java:[82,28] (naming) AbbreviationAsWordInName: Abbreviation in name 'getASubCo
llectionDocumentRef' must contain no more than '1' consecutive capital letters.
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check (default-cli) on project firestore-snippets: You ha
ve 8 Checkstyle violations. -> [Help 1]
```

--- a/samples/snippets/src/main/java/com/example/firestore/Quickstart.java
+++ b/samples/snippets/src/main/java/com/example/firestore/Quickstart.java
@@ -119,7 +119,7 @@ public class Quickstart {
     }
   }
 
-  void runAQuery() throws Exception {
+  void runQuery() throws Exception {
     // [START fs_add_query]
     // asynchronously query for all users born before 1900
     ApiFuture<QuerySnapshot> query =
@@ -177,7 +177,7 @@ public class Quickstart {
 
     // retrieve all users born before 1900
     System.out.println("########## users born before 1900 ##########");
-    runAQuery();
+    runQuery();
 
     // retrieve all users
     System.out.println("########## All users ##########");

--- a/samples/snippets/src/main/java/com/example/firestore/snippets/ManageDataSnippets.java
+++ b/samples/snippets/src/main/java/com/example/firestore/snippets/ManageDataSnippets.java
@@ -412,7 +412,7 @@ class ManageDataSnippets {
     ApiFuture<List<WriteResult>> future = batch.commit();
     // ...
     // future.get() blocks on batch commit operation
-    for (WriteResult result :future.get()) {
+    for (WriteResult result : future.get()) {
       System.out.println("Update time : " + result.getUpdateTime());
     }
     // [END fs_write_batch]

--- a/samples/snippets/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java
+++ b/samples/snippets/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java
@@ -109,7 +109,7 @@ class QueryDataSnippets {
    *
    * @return query
    */
-  Query createAQuery() throws Exception {
+  Query createQuery() throws Exception {
     // [START fs_create_query]
     // Create a reference to the cities collection
     CollectionReference cities = db.collection("cities");
@@ -130,7 +130,7 @@ class QueryDataSnippets {
    *
    * @return query
    */
-  Query createAQueryAlternate() throws Exception {
+  Query createQueryAlternate() throws Exception {
     // [START fs_create_query_country]
     // Create a reference to the cities collection
     CollectionReference cities = db.collection("cities");

--- a/samples/snippets/src/main/java/com/example/firestore/snippets/References.java
+++ b/samples/snippets/src/main/java/com/example/firestore/snippets/References.java
@@ -34,7 +34,7 @@ public class References {
    *
    * @return collection reference
    */
-  public CollectionReference getACollectionRef() {
+  public CollectionReference getCollectionRef() {
     // [START fs_collection_ref]
     // Reference to the collection "users"
     CollectionReference collection = db.collection("users");
@@ -47,7 +47,7 @@ public class References {
    *
    * @return document reference
    */
-  public DocumentReference getADocumentRef() {
+  public DocumentReference getDocumentRef() {
     // [START fs_document_ref]
     // Reference to a document with id "alovelace" in the collection "users"
     DocumentReference document = db.collection("users").document("alovelace");
@@ -60,7 +60,7 @@ public class References {
    *
    * @return document reference
    */
-  public DocumentReference getADocumentRefUsingPath() {
+  public DocumentReference getDocumentRefUsingPath() {
     // [START fs_document_path_ref]
     // Reference to a document with id "alovelace" in the collection "users"
     DocumentReference document = db.document("users/alovelace");
@@ -73,7 +73,7 @@ public class References {
    *
    * @return document reference in a subcollection
    */
-  public DocumentReference getASubCollectionDocumentRef() {
+  public DocumentReference getSubCollectionDocumentRef() {
     // [START fs_subcollection_ref]
     // Reference to a document in subcollection "messages"
     DocumentReference document =

--- a/samples/snippets/src/test/java/com/example/firestore/QuickstartIT.java
+++ b/samples/snippets/src/test/java/com/example/firestore/QuickstartIT.java
@@ -55,7 +55,7 @@ public class QuickstartIT extends BaseIntegrationTest {
     addData();
 
     bout.reset();
-    quickstart.runAQuery();
+    quickstart.runQuery();
     String output = bout.toString();
     // confirm that results do not contain aturing
     assertTrue(output.contains("alovelace"));

--- a/samples/snippets/src/test/java/com/example/firestore/snippets/QueryDataSnippetsIT.java
+++ b/samples/snippets/src/test/java/com/example/firestore/snippets/QueryDataSnippetsIT.java
@@ -53,8 +53,8 @@ public class QueryDataSnippetsIT extends BaseIntegrationTest {
   }
 
   @Test
-  public void testCreateAQuery() throws Exception {
-    Query q = queryDataSnippets.createAQuery();
+  public void testCreateQuery() throws Exception {
+    Query q = queryDataSnippets.createQuery();
     Set<String> result = getResultsAsSet(q);
     Set<String> expectedResults = new HashSet<>(Arrays.asList("DC", "TOK", "BJ"));
     assertTrue(Objects.equals(result, expectedResults));

